answer-transcode-mobile.py

```diff
@@ -38,10 +38,10 @@ def transcode_mobile(video_file, s3_path):
     )
 
 
-def process_task(request, task):
+def process_task(request):
     log.info("video to process %s", request["video"])
     stored_task = fetch_from_graphql(
-        request["mentor"], request["question"], task["task_id"]
+        request["mentor"], request["question"], "transcodeMobileTask"
```

Comment:
you sure about this? cant a mentor have multiple tasks for the same question, say submitting an answer in two tabs at the same time?

Reply:
GraphQL has a unique constraint on the mentor and question fields for this document.